software/samba/en.md

```diff
@@ -35,15 +36,84 @@ sudo systemctl enable --now smb
 # Check whether Samba is running
 sudo systemctl status smb
 
+# Restart Samba manually
+sudo systemctl restart smb
+
 # Stop Samba manually
 sudo systemctl stop smb
 
 # Configure Samba to not start automatically on each boot and immediately stop the service
 sudo systemctl disable --now smb
 ```
 
+### How to start the Web Service Discovery Daemon to enable Windows 7+ Network discovery
```

Comment:
This is a really long header. Maybe something simpler like "Enabling Windows 7+ Network Discovery" then have a short explanation about wsdd?

Reply:
Good idea. Will fix.